include/dao/db_manager.js

```
@@ -152,19 +152,33 @@ module.exports = function DBManagerModule(pb) {
          * defer the reporting of an error until the end.
          */
         this.processIndices = function(procedures, cb) {
+            var self = this;
             if (!util.isArray(procedures)) {
                 cb(new Error('The procedures parameter must be an array of Objects'));
                 return;
             }
 
+            this.dropUnconfiguredIndices(procedures, function(err) {
+                if(util.isError(err)) {
+                    pb.log.error('DBManager: Error occurred during index check/deletion ERROR[%s]', err.stack);
+                }
```

did you mean to silently fail here?